save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/utils/Warning.kt

```diff
@@ -12,11 +12,13 @@ import org.cqfn.save.core.plugin.ResourceFormatException
  * @property message warning text
  * @property line line on which this warning occurred
  * @property column column in which this warning occurred
+ * @property fileName file name
  */
 data class Warning(
     val message: String,
     val line: Int?,
     val column: Int?,
+    val fileName: String?,
```

Comment:
I think it shouldn't be nullable: we will always know, from which file we have got the warning